widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/FolderTree.java

```diff
@@ -150,12 +150,13 @@ public void onResponseReceived( Request request, Response response ) {
 
   @NonNull
   private String getServiceBaseUrl() {
-    return getFullyQualifiedURL() + "plugin/scheduler-plugin/api/generic-files/folders/";
+    return getFullyQualifiedURL() + "plugin/scheduler-plugin/api/generic-files/files/";
```

Comment:
All good except the `/files` path segment in the URLs, as noted in the related PR.